demos/signal-protocol/python/saw_helpers.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional, Tuple
+
+from saw_client.llvm import Contract, FreshVar, LLVMType, SetupVal
+
+
+def ptr_to_fresh(spec : Contract, ty : LLVMType, name : Optional[str] = None) -> Tuple[FreshVar, SetupVal]:
```

Comment:
We should probably add this and similar things to the `saw_client` library, since they're so commonly used.

Reply:
If I understand correctly, this is what GaloisInc/saw-script#253 is tracking, right? I agree that `ptr_to_fresh` (or something with a similar name) would be nice to have, and it would be much easier to maintain a Python prelude than a SAWScript prelude.

Reply:
Yep, exactly.